src/easy-peasy.js

```diff
@@ -233,6 +233,13 @@ export const createStore = (model, options = {}) => {
                 )
                 return result
               })
+              .catch(err => {
+                references.dispatch({
+                  type: `${name}(failed)`,
+                  payload: err,
+                })
```

Comment:
Maybe this should throw to propagate the error?

```javascript
              .catch(err => {
                references.dispatch({
                  type: `${name}(failed)`,
                  payload: err,
                })
                throw err;
              })
```

Which would result in a test result of:

```javascript
    expect(trackActions.actions).toEqual([
      { type: '@thunk.foo.doSomething(started)', payload },
      { type: '@thunk.foo.error(started)', payload: undefined },
      { type: '@thunk.foo.error(failed)', payload: undefined },
      { type: '@thunk.foo.doSomething(failed)', payload },
    ])
```

Reply:
True! Good catch. Thanks @ctrlplusb